packages/sx05re/emuelec/config/emuelec/scripts/show_splash.sh

```diff
@@ -58,18 +69,18 @@ fi
 
 if [[ -f "/storage/.config/emuelec/configs/novideo" ]] && [[ ${VIDEO} != "1" ]]; then
 	if [ "$PLATFORM" != "intro" ]; then
-		(
+		#(
```

Comment:
Did you try this with both images and videos? by removing the subshell the script cannot continue until mpv ends.

Reply:
Yes, I tested it with both images and videos . becasuse I need it to play video splash, so i have to wait until video ends. If video is a little longer, game would start when video is still being played.